oak/proto/manager.proto

```diff
@@ -41,6 +41,7 @@ message Node {
     GrpcServerNode grpc_server_node = 3;
     WebAssemblyNode web_assembly_node = 4;
     StorageProxyNode storage_node = 5;
+    HttpClientProxyNode http_client_node = 6;
```

Comment:
I would just call it `HttpClientNode` (no "proxy").

Reply:
Unfortunately this causes `build_server_asylo` to fail as the protobuff generated class collides with `oak/server/http_client_node.h`'s `HttpClientNode`. I agree there should be a better solution/naming here though

Reply:
That's annoying, maybe we should use a separate namespace, but for now I guess you could call the C++ class OakHttpClientNode (though I think that's not great, that's what namespaces are for, arguably :) )

Reply:
In fact, could you create a GitHub issue for this? I guess we could move all the C++ node implementations under `oak::nodes`, and leave the proto namespaces as they are.

@daviddrysdale WDYT?

Reply:
Moving the proto-generated stuff to a separate namespace would make sense in theory (and I've hit similar problems in the past and just done the lazy `Oak...` prefix).

However, I don't think it's worth putting any effort into it &ndash; I'm expecting to do away with the static Node configuration in favour of dynamically created Nodes everywhere, which will totally change the `ApplicationConfiguration` protobuf anyhow.